### PR TITLE
add Grafana to the client scheme

### PR DESCRIPTION
**Pull request by elenagerman**
add Grafana to the client scheme

### CODE REVIEW

**Comment by elenagerman** [comment]
@trewest @cdvultur @achuzhoy @kononovn PTAL, it's a quite urgent

File: pkg/clients/clients.go [hunk 1]
```diff
@@ -57,6 +57,7 @@ import (
 	coreV1Client "k8s.io/client-go/kubernetes/typed/core/v1"
 
 	nvidiagpuv1 "github.com/NVIDIA/gpu-operator/api/v1"
+	grafanaV1Alpha1 "github.com/grafana-operator/grafana-operator/v4/api/integreatly/v1alpha1"
```
Comment (cdvultur) [inline pkg/clients/clients.go:60]: I think the client name should reflect that is using the [v4](https://github.com/grafana-operator/grafana-operator/tree/v4) branch.

Also it looks like that branch is on go 1.18 . @kononovn will that cause issues ?

**Comment by kononovn** [comment]
I tested this PR. Clinet works after those changes.

**Review by kononovn — APPROVED** [review]
lgtm

**Review by trewest — APPROVED** [review]
/lgtm

**Pull request merged**